### DISCUSSION TITLE
#59 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Thegaram** [tog-ETH-er/togETHer#44] [General]
Agenda:

- Community Updates
- [Shanghai Upgrade event](https://www.shanghai-upgrade.org/) vote results
- TogETHer offline meetup: April Beijing, Guangzhou, Shenzhen, Hong Kong
- Fireside Chat

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/wba-mxnt-bsu

**Comment by liushooter** [comment]
## rebase

- web3极客日报 最近 14 期 http://daily.rebase.network/
-  6份招聘 https://github.com/rebase-network/who-is-hiring/issues

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

11个PR #934 ～ #945

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了4场分享会

- 《通过手写OPCode的方式来节省gas：以sudoswap为例》by Comcat
  - https://www.youtube.com/watch?v=r0_-c144EFY
  - https://www.youtube.com/watch?v=vt7VraNpsgM

- 《密码学系列课程》by lynndell
  1. 对称加密与哈希函数 https://www.youtube.com/watch?v=whdoh1RSh94
  2. 公钥加密与数字签名 https://www.youtube.com/watch?v=Ow4ABckI9yE
 
- 《Optimism Bedrock》by paco
  - https://www.youtube.com/watch?v=o4MyjQiJe2U
  - https://www.youtube.com/watch?v=l8ZTKtDUCTw
  - https://www.youtube.com/watch?v=7yngjH_VbWU

### 密码学系列课程

课程目录

密码学基础系列

1. 对称加密与哈希函数
2. 公钥加密与数字签名
3. 密码协议：承诺、零知识证明、密钥协商
4. 同态加密

ECDSA 多签系列

1. Li17 两方签名
2. GG18 多方签名
3. GG20 多方签名
4. CMP20 多方签名
6. DKLs18 两方/20 多方签名
7. Schnorr/EdDSA 多方签名

zk 系列

1. Groth16 证明系统
2. Plonk 证明系统
3. UltraPlonk 证明系统
4. SHA256 查找表技术
5. Halo2 证明系统
6. zkSTARK 证明系统

**Comment by Thegaram** [comment]
## Crypto Tech Night

CTN#21: Shanghai Upgrade on 30th March.
- Speakers: Luozhu Zhang, Ping Chen, 0xAA.
- Tech track of https://www.shanghai-upgrade.org
- Call to help: Speaker about Danksharding/4844 or future Ethereum roadmap.

**Comment by Stamp9** [comment]
## 原语里弄

研究项目进行中。

**Comment by stephanieeee-ho** [comment]
### ECN
上海升级相关文章：
1. 验证者必读：ETH 提款 FAQ
    https://www.ethereum.cn/Eth2/withdrawals-faq
2. Goerli 测试网 Shapella 升级公告
    https://www.ethereum.cn/Eth2/goerli-shapella-announcement
3. 理解以太坊质押提款
    https://www.ethereum.cn/Eth2/understanding-withdrawals
4. 以太坊的上海升级将如何影响 Staking 的未来
    https://www.ethereum.cn/Eth2/The-Future-of-Staking

**Comment by 0x68616f79756e** [comment]
**ETHPlanet**
1.ETHPlanet meetup

2.协办ETHBeijing
https://www.ethbeijing.xyz/

3.协办EDCON
https://edcon.io/

4.帮助协调Web 3 Festival
https://www.web3festival.org/
